.github/actions/setup-capnproto/action.yml

```diff
@@ -0,0 +1,24 @@
+name: "Setup Capnproto"
+description: "Sets up capnproto for the current platform"
+inputs:
+  target:
+    description: "Compilation target"
+    required: true
```

Comment:
Instead of passing in the compilation target can we instead use [`runner.os`](https://docs.github.com/en/actions/learn-github-actions/variables#detecting-the-operating-system)? Also, I think that this is the compilation host and not target.